src/style/blocks/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
```

Comment:
your card should have padding for all sides
So then, you will just add margin-bottom for children to make an indent between elements